src/OpenSpaceNet.cpp

```diff
@@ -276,6 +281,7 @@ void OpenSpaceNet::initFeatureSet()
     VectorOpenMode openMode = args_.append ? APPEND : OVERWRITE;
 
     featureSet_ = make_unique<FeatureSet>(args_.outputPath, args_.outputFormat, args_.layerName, definitions, openMode);
+    featureSet_->setProjection(sr_.wkt());
```

Comment:
I pulled master into my branch and now setting the spatial reference is done in `createLayer` rather than `setProjection`.